.github/workflows/default.yml

```diff
@@ -0,0 +1,154 @@
+---
+name: AnsibleCI
+
+on:
+  push:
+  pull_request:
+  # schedule:  # run weekly, every monday 01:00
+  #   - cron: '0 1 * * 1'
```

Comment:
A `schedule` will only run when the file is on the default branch. It wont run yet while still a pr / branch.
If it should be running in the future this can be commented in already. If it should not be used it should be removed completely rather than just being a comment.

Also a minor idea: Monday is probably the most crowded day for CI. Maybe do it on Sunday evening?

Reply:
I don't see why running a full deployment on a cron base would make any sense :D
So removing it completely is the best option.

Reply:
Maybe new releases of BBB packages might introduce changes that might break this role. But as most configs are overwritten anyway I doubt that might have a big impact.

Thats the only thing that came to my mind when thinking about the `schedule` so far.